conf/messages

```diff
@@ -2,4 +2,6 @@ error.postcode.invalid = Please enter a valid postcode
 error.utr.invalid = Please enter a valid UTR
 error.telephone.invalid = Please enter a valid telephone number
 error.no.ampersand = Agency name cannot contain "&"
-
+error.name.invalid = "This field is mandatory and is limited to 40 alphanumeric characters (A-Z, a-z 0-9 \\-,.)"
```

Comment:
I don't think these 3 messages need to mention "This field is mandatory" because if you don't enter a value you don't see one of these three messages, you see the "This field is required"